web/pages/create-dashboard.tsx

```diff
@@ -0,0 +1,94 @@
+import { useState } from 'react'
+import { Spacer } from 'web/components/layout/spacer'
+import { Page } from 'web/components/page'
+import { Title } from 'web/components/title'
+import Textarea from 'react-expanding-textarea'
+
+import { TextEditor, useTextEditor } from 'web/components/editor'
+import { createDashboard } from 'web/lib/firebase/api'
+import clsx from 'clsx'
+import { useRouter } from 'next/router'
+import { Dashboard, MAX_DASHBOARD_NAME_LENGTH } from 'common/dashboard'
+import { dashboardPath } from 'web/lib/firebase/dashboards'
+
+export default function CreateDashboard() {
```

Comment:
You might consider moving this file to `pages/dashboard/create`. That url structure makes a little more sense to me and the file would be colocated with the dashboard index.